contracts/TokenBacking.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.5.0 <0.6.0;
+
+import "./common/Ownable.sol";
+import "./common/Pausable.sol";
+import "./interfaces/IRelay.sol";
+import "./interfaces/IERC20.sol";
+import "./interfaces/ISettingsRegistry.sol";
+import { ScaleStruct } from "./common/Scale.struct.sol";
+
+import "./common/Scale.sol";
+import "./common/SafeMath.sol";
+
+pragma experimental ABIEncoderV2;
+
+contract TokenBacking is Ownable, Pausable {
```

Comment:
```suggestion
contract TokenIssuing is Ownable, Pausable {
```

This is used for verify lock proof and mint token, should call it `TokenIssuing`